kivy/uix/recycleview/__init__.py

```diff
@@ -8,6 +8,9 @@
 large data sets. It aims to prevent the performance degradation that can occur
 when generating large numbers of widgets in order to display many data items.
 
+.. warning::
+    Because RecycleView reuses widgets, any state change to a single widget will stay with that widget, even if the data assigned to it by the RecycleView changes.
```

Comment:
```suggestion

    Because :class:`RecycleView` reuses widgets, any state change to a single
    widget will stay with that widget as it's reused, even if the
    :attr:`~RecycleView.data` assigned to it by the :class:`RecycleView`
    changes. Unless the complete state is tracked in :attr:`~RecycleView.data`
    (see below).
```